libs/ui-lib-tests/cypress/support/interceptors.ts

```diff
@@ -103,6 +107,27 @@ const mockInfraEnvResponse: HttpRequestInterceptor = (req) => {
   }
 };
 
+const mockCustomManifestResponse: HttpRequestInterceptor = (req) => {
+  const fixtureMapping = getScenarioFixtureMapping();
+  if (fixtureMapping?.manifests) {
+    req.reply(fixtureMapping.manifests);
+  }
+};
+
+const mockCustomManifestFileResponse: HttpRequestInterceptor = (req) => {
+  const fixtureMapping = getScenarioFixtureMapping();
+  if (fixtureMapping?.manifestContent) {
+    req.reply(fixtureMapping.manifestContent);
+  }
+};
+
+const mockCustomManifestFileResponse2: HttpRequestInterceptor = (req) => {
+  const fixtureMapping = getScenarioFixtureMapping();
+  if (fixtureMapping?.manifestContent) {
+    req.reply(fixtureMapping.manifestContent);
+  }
+};
+
```

Comment:
What's the difference between these two?

Reply:
Are equals. I've deleted the second one.